application/backend/src/main/resources/apps/typerefinery/components/widgets/map/clientlibs/behaviour.js

```diff
@@ -0,0 +1,14 @@
+window.Typerefinery = window.Typerefinery || {};
+window.Typerefinery.Components = Typerefinery.Components || {};
+window.Typerefinery.Components.Widgets = Typerefinery.Components.Widgets || {};
+window.Typerefinery.Components.Widgets.Map = Typerefinery.Components.Widgets.Map || {};
+window.Typerefinery.Components.Widgets.Map.LeafletMap = Typerefinery.Components.Widgets.Map.LeafletMap || {};
+console.log("behaviour.js")
```

Comment:
Please remove the console which is added for debugging purposes.

Reply:
Done